native/ex_tokenizers/src/util.rs

```diff
@@ -0,0 +1,11 @@
+use rustler::NifUntaggedEnum;
+
+#[derive(NifUntaggedEnum)]
+pub enum DetailValue {
```

Comment:
If we change that `HashMap` for an encoded Map as `Term`, we can get rid of this enum, since all the primitives can be encoded already.